recipes/libcf/build.sh

```diff
@@ -0,0 +1,15 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+export LFLAGS="-fPIC"
+
+CONDA_LST=`conda list`
+if [[ ${CONDA_LST}'y' == *'openmpi'* ]]; then
```

Comment:
The discussion of how to express which mpi implementation is used is mainly [here](https://github.com/conda-forge/staged-recipes/pull/1501). Features should work for this, it's mainly a question of deciding precisely how those features should be defined / tracked.

If there does need to be logic in build.sh, I'm not sure that such things are passed down to the build environment. Are they?